setup.cfg

```diff
@@ -8,32 +8,32 @@ test=pytest
 addopts = --ignore=scripts
 
 [flake8]
-ignore = E501, W503
+ignore = W503, E203, N806
+max-line-length = 88
 per-file-ignores =
     pyuvdata/tests/*.py: D
     pyuvdata/*/tests/*.py: D
-    docs/*.py: D
+    docs/*.py: D,A
     setup.py: D
-# remove the following lines as they get converted to numpy style docstrings:
-    pyuvdata/uvdata/aipy_extracts.py: D
-    pyuvdata/uvbase.py: D
-    pyuvdata/uvcal/uvcal.py: D
-    pyuvdata/uvflag/uvflag.py: D
+# remove the D from following lines as they get converted to numpy style docstrings:
+# remove the A from following lines as builtin shadowing is fixed:
+# remove the N802, N803 from following lines as functions and input variables are renamed to pep8 style:
+    pyuvdata/uvdata/aipy_extracts.py: D,A
+    pyuvdata/uvcal/uvcal.py: D,RST
+    pyuvdata/uvflag/uvflag.py: D,A
+    pyuvdata/uvflag/tests/test_uvflag.py: D,A
+    pyuvdata/uvdata/uvdata.py: N802
+    pyuvdata/uvbeam/mwa_beam.py: N802
+    pyuvdata/utils.py: N802, N803
+    pyuvdata/tests/test_utils.py: D,N802
+    pyuvdata/tests/__init__.py: D,N802
 docstring-convention = numpy
-select = C,E,W,T4,B9,F,D
+select = C,E,W,T4,B9,F,D,A,N,RST,B
 # List of other checks to consider adding:
-# move to a max line length rather than ignoring E501
-# max-line-length = 88
 # it's recommended to have max-complexity ~ 18
 # max-complexity = 18
-# flake8-quotes
-# flake8-comprehensions
-# flake8-black
-# flake8-builtins
 # flake8-eradicate
-# pep8-naming
 # flake8-isort
-# flake8-rst-docstrings
 # flake8-rst
 # darglint
```

Comment:
I think this is the one that checks if all arguments are in your docstring -- but it doesn't support numpydoc yet

Reply:
yep, figured I'd leave it there so we'd find it again in the future.